srcpkgs/The-Powder-Toy/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'The-Powder-Toy'
+pkgname=The-Powder-Toy
+version=94.1
+revision=1
+build_style=scons
+makedepends="SDL2-devel lua51-devel fftw-devel bzip2-devel zlib-devel"
+short_desc="Falling sand physics sandbox, simulates air pressure, velocity & heat"
+maintainer="Illia Shestakov <ishestakov@airmail.cc>"
+license="GPL-3.0-or-later"
+homepage="https://powdertoy.co.uk/"
+distfiles="https://github.com/The-Powder-Toy/${pkgname}/archive/v${version}.tar.gz"
+checksum=9770c0732c97e4e89a1e53330c969e9675595f78e6673c519c3724d6df591e72
+CPPDEFINES=IGNORE_UPDATES
+
+case "$XBPS_TARGET_MACHINE"
+	in x86_64*|i686*)
+		;;
+	*)
+		make_build_args="--no-sse";;
+esac
+
+do_install() {
+	case "$XBPS_TARGET_MACHINE" in
+	x86_64*)
+		vbin build/powder64 powder;;
+	i686*)
+		vbin build/powder powder;;
+	*)
+		vbin build/powder-legacy powder;;
```

Comment:
Maybe set the output option instead https://github.com/The-Powder-Toy/The-Powder-Toy/blob/master/SConscript#L571

Reply:
You're right, it works